### PR TITLE
Add Dart support

### PR DESCRIPTION
**Pull request by jcs090218**

### CODE REVIEW

**Comment by jcs090218** [comment]
@meain Can you review this? Thanks!

Note: This should be good to merge after #198.

File: queries/dart/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    (final_builtin)
+    "abstract"
+    "async"
+    "async*"
+    "covariant"
+    "dynamic"
+    "external"
+    "static"
+    "sync*"
+] @type.qualifier
```
[the remark this answers is not in the document]
Comment (jcs090218) [reply queries/dart/highlights.scm]: I've updated to use `type` instead.

File: queries/dart/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,269 @@
+(dotted_identifier_list) @string
+
+; Methods
+; --------------------
+(super) @function
+
+; TODO: add method/call_expression to grammar and
+; distinguish method call from variable access
+(function_expression_body (identifier) @function)
```
[the remark this answers is not in the document]
Comment (jcs090218) [reply queries/dart/highlights.scm]: 👍 Done.

**Comment by meain** [comment]
Might wanna rebase this on top of `master` for the CI to work.

**Comment by jcs090218** [comment]
Okay, the GitHub website doesn't rebase (green button); they merge instead. I hope that's okay. 😕

**Comment by jcs090218** [comment]
Do I need to make changes to the CHANGELOG? 🤔

**Pull request merged**